### PR TITLE
Set packages version in setup.py

### PR DESCRIPTION
**Pull request by florian-vuillemot**

### CODE REVIEW

**Review by jrobinAV — DISMISSED** [review]
What s the reason you changed the GitHub actions?

**Comment by florian-vuillemot** [comment]
.github/workflows/doc.yml => is not working anymore because "user-to-notify" is not a valid argument
.github/workflows/coverage.yml & .github/workflows/test.yml => we do not need SSH key anymore because taipy-core is public :)

File: Pipfile
Comment (joaoandre-avaiga) [inline Pipfile:12]: This may be a problem. At the time I had restricted the flask version because of the requirements of taipy gui, not sure if they updated their version.

File: tests/test_datanode.py
Comment (joaoandre-avaiga) [inline tests/test_datanode.py:79]: I don't understand why add the json param to this function.

Comment (florian-vuillemot) [reply tests/test_datanode.py:79]: Else tests failed with the bad request status return by Flask :/

File: Pipfile
Comment (florian-vuillemot) [reply Pipfile:12]: yes, the new version for taipy-gui is Flask>=2.0 now! But I will wait for them approval ^^

**Review by joaoandre-avaiga — DISMISSED** [review]

**Review by jrobinAV — DISMISSED** [review]
LGTM

**Review by joaoandre-avaiga — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**
